<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BusinessRulesDialog</class>
 <widget class="QDialog" name="BusinessRulesDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1170</width>
    <height>719</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Alarm/Event Rules</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="0,0,0">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <layout class="QVBoxLayout" name="mainLayout">
     <property name="spacing">
      <number>16</number>
     </property>
     <property name="leftMargin">
      <number>16</number>
     </property>
     <property name="topMargin">
      <number>16</number>
     </property>
     <property name="rightMargin">
      <number>16</number>
     </property>
     <property name="bottomMargin">
      <number>0</number>
     </property>
     <item>
      <layout class="QHBoxLayout" name="searchLayout">
       <item>
        <widget class="QLineEdit" name="filterLineEdit">
         <property name="placeholderText">
          <string notr="true">&lt; filter by camera... &gt;</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="clearFilterButton">
         <property name="text">
          <string>X</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="topLayout">
       <item>
        <widget class="QPushButton" name="addRuleButton">
         <property name="text">
          <string>Add</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteRuleButton">
         <property name="text">
          <string>Delete</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="eventLogButton">
         <property name="text">
          <string>Event log</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/skin/tree/layout.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="advancedButton">
         <property name="text">
          <string>Advanced...</string>
         </property>
         <property name="icon">
          <iconset>
           <normalon>:/skin/titlebar/dropdown_hovered.png</normalon>
          </iconset>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QnTableView" name="tableView">
       <property name="editTriggers">
        <set>QAbstractItemView::AllEditTriggers</set>
       </property>
       <property name="showDropIndicator" stdset="0">
        <bool>false</bool>
       </property>
       <property name="alternatingRowColors">
        <bool>true</bool>
       </property>
       <property name="selectionMode">
        <enum>QAbstractItemView::SingleSelection</enum>
       </property>
       <property name="selectionBehavior">
        <enum>QAbstractItemView::SelectRows</enum>
       </property>
       <property name="showGrid">
        <bool>false</bool>
       </property>
       <property name="sortingEnabled">
        <bool>true</bool>
       </property>
       <attribute name="horizontalHeaderCascadingSectionResizes">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderShowSortIndicator" stdset="0">
        <bool>true</bool>
       </attribute>
       <attribute name="horizontalHeaderStretchLastSection">
        <bool>true</bool>
       </attribute>
       <attribute name="verticalHeaderVisible">
        <bool>false</bool>
       </attribute>
      </widget>
     </item>
     <item>
      <widget class="QnBusinessRuleWidget" name="detailsWidget" native="true"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QnTableView</class>
   <extends>QTableView</extends>
   <header location="global">ui/widgets/common/table_view.h</header>
  </customwidget>
  <customwidget>
   <class>QnBusinessRuleWidget</class>
   <extends>QWidget</extends>
   <header location="global">ui/widgets/business/business_rule_widget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../../../x64/build/client.qrc"/>
 </resources>
 <connections/>
</ui>
